.travis.yml

```diff
@@ -1,10 +1,10 @@
 ---
 language: node_js
 node_js:
-  - "node"
-  - "lts/*"
```

Comment:
since LTS is now 10 instead of 8, did this still test the latest version of 8?  (and it was missing 9). I like the symmetry of just listing the numbers.

Also, I explicitly stopped testing 8.0.0 (sorry), because a dev package no longer supports 8.0.0 (eslint?), so yarn install fails.

Reply:
works for me!